pkgs/applications/science/math/pari/default.nix

```diff
@@ -82,7 +72,6 @@ stdenv.mkDerivation rec {
            of the GP language.
     '';
     homepage    = "http://pari.math.u-bordeaux.fr";
-    downloadPage = "http://pari.math.u-bordeaux.fr/download.html";
```

Comment:
Why? I think homepage doesn't have direct tarball link, and the download page does…

Reply:
Well, I think it is a bit superfluous, and we don't use "updaters" here. But yes, on the other hand the direct link is not so obvious.